test/smoke/smoke_test.go

```diff
@@ -309,6 +323,15 @@ var _ = Describe("Smoke test", func() {
 				return siteIndexerStatus
 			}, deployment.GetTimeout(), PollInterval).Should(Equal(siteIndexerMap))
 
+			// Verify MC Pod is Ready
+			testenv.MCPodReady(testenvInstance.GetName(), deployment)
+
+			// Verify RF SF is met
+			Eventually(func() bool {
+				rfSfStatus := testenv.CheckRFSF(testenvInstance.GetName(), deployment.GetName())
+				testenvInstance.Log.Info("Verifying RF SF is met", "Status", rfSfStatus)
+				return rfSfStatus
```

Comment:
Looks like this code is repetition in all the above 3 places. Wondering if this function can be defined explicitly and called from `Eventually()`?